src/tools/cephfs/top/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+include(Distutils)
+distutils_install_module(cephfs-top)
+
+if(WITH_TESTS)
+  include(AddCephTest)
+  add_tox_test(cephfs-top)
```

Comment:
why? i cannot find `tox.ini` under this directory.

Reply:
that's because I missed adding it to the commit -- exaplains the tox failures.

Will push an update.